README.md

```diff
@@ -9,12 +9,21 @@ Horarium is fully based on an asynchronous work model, it allows you to run hund
 
 Horarium supports .NET Core/netstandard 2.0 and .NET Framework 4.6.2 and later.
 
+Support Databases
+
+| Database   | Support                                                            |
+| ---------- | ------------------------------------------------------------------ |
+| MongoDB    | Yes                                                                |
+| In Memory  | No [#5](https://github.com/TinkoffCreditSystems/Horarium/issues/5) |
```

Comment:
Let's write `Not yet` instead of `No` 🙂 .

Reply:
done